modules/monitoring-enabling-the-platform-alertmanager-instance-for-user-defined-alert-routing.adoc

```diff
@@ -11,6 +11,7 @@ You can allow users to create user-defined alert routing configurations that use
 .Prerequisites
 
 * You have access to the cluster as a user with the `cluster-admin` cluster role.
+* A cluster administrator has enabled monitoring for user-defined projects.
```

Comment:
* `monitoring for user-defined projects was enabled.` this would require changes throughout the doc. I am not necessarily against that change :) but I remember discussing this a while back and there was a reason why we included it this way `A cluster administrator has enabled...`  but I don't remember the reason. So let's wait for other opinions.

* `...` You are 100% right about this. In the doc guidelines it is specified to add  ellipsis in this form: `# ...` So I will change it here (I will take a look at the rest of our examples in other PRs as I will be looking at each of our procedures anyway as part of the restructuring effort :+1: )